bedrock/mozorg/templates/mozorg/home/home-new.html

```diff
@@ -30,8 +30,8 @@
   {{ css_bundle('protocol-newsletter') }}
   {{ css_bundle('home-new') }}
 
-  {% if switch('monitor-plus-banner') and LANG == 'en-US' %}
-    {{ css_bundle('monitor-banner')}}
```

Comment:
**Issue:** at the bottom of the page I can still see the un-styled monitor banner, as the HTML include is still in the page:

![image](https://github.com/mozilla/bedrock/assets/400117/a4d2b7dd-927e-440b-8858-fd0722b5e5bc)

Reply:
I think we're still investing in Monitor Plus, but let me double-check on things w/ Dan and Emma before we go ahead and remove it.

Reply:
Checked with Emma, and she suggested we keep the banner code in Bedrock in case the Monitor team asks for it to be used once things are sorted out with the new provider